src/Powercord/apis/settings/index.js

```diff
@@ -78,27 +85,22 @@ module.exports = class Settings extends API {
       panelContents = null;
     }
 
-    const h2 = React.createElement(getComponentByDisplayName('FormTitle'), { tag: 'h2' }, title);
-    return React.createElement(getComponentByDisplayName('FormSection'), {}, h2, panelContents);
+    const h2 = React.createElement(FormTitle, { tag: 'h2' }, title);
+    return React.createElement(FormSection, {}, h2, panelContents);
   }
 
   // @todo: Discord settings sync
   async upload () {
-    if (!powercord.account || !this.settings['pc-general'].get('settingsSync', false)) {
+    if (!powercord.account || !this.store.settings['pc-general'].get('settingsSync', false)) {
```

Comment:
Change `this.store.settings['pc-general'].get('settingsSync', false)` to `this.store.settings.getSetting('pc-general', 'settingsSync', false)` as the `get` method was removed (since 4bc68e9) and is now deprecated.

**Note**: The same applies to [line 131](https://github.com/powercord-org/powercord/blob/10e86cd5d4cf39e13d3b4bd85f6a6be6753f5827/src/Powercord/apis/settings/index.js#L131) where the old method remains still in use.

Reply:
yh aware of this one and fixed locally, just didn't pushed lol